eDNAExplorer_Alpha.R

```diff
@@ -29,7 +27,7 @@ Sys.setenv("AWS_ACCESS_KEY_ID" = "<insert key>",
 #* @param EnvironmentalParameter:string Environmental variable to analyze against alpha diversity
 #* @param AlphaDiversity:string Alpha diversity metric
 #* @get /Tronko_Input
-function(First_Date,Last_Date,Marker,Num_Mismatch,TaxonomicRank,CountThreshold,FilterThreshold,EnvironmentalParameter,AlphaDiversity){
+alpha <- function(Object_Name,First_Date,Last_Date,Marker,Num_Mismatch,TaxonomicRank,CountThreshold,FilterThreshold,EnvironmentalParameter,AlphaDiversity){
```

Comment:
@levisimons the function needs to be defined like this to be able to be imported from the api.R file